calico_node/rpm/calico-dockerless-confd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=confd for Dockerless Calico
+Requires=calico-dockerless.service
+Requires=calico-dockerless-bird.service
+After=calico-dockerless.service
+Before=calico-dockerless-bird.service
```

Comment:
Does this work?

`calico-dockerless-bird.service` is both a "Requires" and "Before"?

Maybe it does - I don't know enough about how systemd handles this.